interfaces/console/src/components/SiteConfigurationStepperDialog/index.tsx

```diff
@@ -0,0 +1,359 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present, Ukama Inc.
+ */
+
```

Comment:
This file contains lot of code, Is it possible we can divide it into small parts.